src/components/shared/CircleImage/index.js

```diff
@@ -67,9 +67,9 @@ const CircleImage = props => {
 
   return (
     <Avatar
-      backgroundColor={backgroundColor}
+      backgroundcolor={backgroundColor}
```

Comment:
Change the prop name where is it used as well.

Reply:
@akshatnitd Made the changes